orm.go

```diff
@@ -753,7 +754,19 @@ func OptFieldTypeSet(cacheType CacheType, cacheSize int) FieldOption {
 }
 
 // OptFieldTypeInt adds an integer field.
-func OptFieldTypeInt(min int64, max int64) FieldOption {
+// No arguments: min = min_int, max = max_int
+// 1 argument: min = limit[0], max = max_int
+// 2 or more arguments: min = limit[0], max = limit[1]
+func OptFieldTypeInt(limits ...int64) FieldOption {
+	min := int64(math.MinInt64)
+	max := int64(math.MaxInt64)
+	// note that we are not able to raise an error if len(limits) > 2
+	if len(limits) > 0 {
+		min = limits[0]
+	}
+	if len(limits) > 1 {
+		max = limits[1]
+	}
```

Comment:
I'd like to at least raise the suggestion of panicing if more than 2 args are passed. Any thoughts?

Reply:
Updated `OptFieldTypeInt` to panic in that case.